docs/docs/user-guides/authentication.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: default
+title: Authentication
+nav_order: 4
+parent: User Guides
+---
+---
+
+Firezone provides the ability to require authentication to establish VPN connections.
+
+## Web Authentication (default)
+
+By default Firezone will use the user's email address and password to authenticate
+their VPN session. You can set the session length to a minimum of `Never` and a
```

Comment:
`Never` is actually the maximum session length -- indefinitely. Might reword to something like

> You can set the session length to a minimum of `1 hour` and maximum of `90 days`. Setting this to `Never` disables this setting allows VPN sessions indefinitely. This is the default.

Reply:
made the change